react/features/connection-stats/components/ConnectionStatsTable.js

```diff
@@ -33,6 +33,12 @@ class ConnectionStatsTable extends Component {
          */
         bitrate: PropTypes.object,
 
+        /**
+         * The number of bridges (aka media servers) currently used in the
+         * conference.
+         */
+        bridgeCount: PropTypes.string,
```

Comment:
The count is really a string and not a number?

Reply:
Haha, nice attention to detail. I had it as a number, but there was an error at some point and I took the lazy way out :)
I'll make it a number and make sure that that's what the lib emits.